README.md

```diff
@@ -48,15 +48,29 @@ We use `playwright` to do integration tests. You can run them with the following
 
 3. Run the integration tests with `npx playwright test`
 
+## Tips for Testing
+
+### UI Mode
+
+Run with `npx playwright test --ui` to get a nice UI to see your tests with.
+
+https://playwright.dev/docs/running-tests#run-tests-in-ui-mode
+
+### Headed Mode
```

Comment:
What's the difference between UI and Headed?

Reply:
Headed shows a browser window, and quits when its done. It's more for just seeing _exactly_ what's happening if you need to. UI shows a custom Playwright UI with a few sidebars and tools that make it easier to interact with.